.github/workflows/test.yml

```diff
@@ -35,82 +35,68 @@ on:
       - "cmd/**"
 
 jobs:
-  dump_contexts_to_log:
+  dump-contexts-to-log:
```

Comment:
It's better to put a more specific file name like `unit-test.yaml`?

Reply:
Thank you for your suggestion 🙏 
I have fixed it in the following commit.
https://github.com/vdaas/vald/pull/2141/commits/aeb816c91382d8c0efc9de2e19d0bb23e8411b10